### PR TITLE
Refactor: Remove Executable Permission and Codebase Refactoring

### PR DESCRIPTION
**Pull request by DestinedCodes**
​
## Description
In this pull request, I've made the following changes to the codebase:

Removed executable permission from all *.py files.
Initialize app from this module
Ensure create tables is run once
Adjust Video schema and some routes
Conducted extensive codebase refactoring.

​
## Related Issue (Link to linear ticket)
No Ticket or issue is open for this 

​
## Motivation and Context
The changes in this Pull Request are needed to improve the efficiency of the existing API. The refactor ensures a neater and efficient codebase although some endoint were greatly changed and new endpints added they all are to make the logic better 

​
## How Has This Been Tested?
As at when this pull request is opened, it's still under development.

​
## Screenshots (if appropriate - Postman, etc):
​
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by DestinedCodes** [comment]
The both endpints works well since the main logic didn't change from what we have before the start-recording route is tested and working. But upload route needs to be tested with base64 data. 

Screenshot_20231007-200549

File: app/routes/video_routes.py [hunk 1]
```diff
@@ -1,126 +1,72 @@
...
-        process_video,
-        video_data.id,
-        file_location,
-        unique_filename,
-    )
-
-    return {
-        "msg": "Video uploaded successfully and is being processed!",
-        "video_data": video_data,
+    Response = {
```
Comment (zxenonx) [inline app/routes/video_routes.py:48]: not a pythonic way to name a variable.

Should be "response"

File: app/routes/video_routes.py [hunk 2]
```diff
@@ -1,126 +1,72 @@
...
 ):
     """
     Uploads a video blob to the server.
 
     Args:
         background_tasks (BackgroundTasks): The background tasks object.
-        video_blob (VideoBlob): The video blob data.
-        db (Session, optional): The database session. Defaults to
-            Depends(get_db).
+        Video_blob (VideoBlob): The video blob data.
```
Comment (zxenonx) [inline app/routes/video_routes.py:67]: Should be "video_data" right?

File: app/routes/video_routes.py [hunk 3]
```diff
@@ -165,35 +110,37 @@ def upload_video_blob(
         #     video_blob.filename,
         # )
 
-        return {
-            "msg": "Video blobs received successfully and video is being processed!",
-            "video_data": video_data,
+        Response = {
```
Comment (zxenonx) [inline app/routes/video_routes.py:113]: "response"

File: app/routes/video_routes.py [hunk 4]
```diff
@@ -165,35 +110,37 @@ def upload_video_blob(
...
-def get_videos(username: str, db: Session = Depends(get_db)):
+@router.get("/videos/user/{user_id}")
+def get_videos(user_id: str, db: Session = Depends(get_db)):
     """
-    Returns a list of videos associated with the given username.
+    Returns a list of videos associated with the given user_id.
     Parameters:
         username (str): The username for which to retrieve the videos.
-        db (Session): The database session.
+        Db (Session): The database session.
```
Comment (zxenonx) [inline app/routes/video_routes.py:132]:
```suggestion
        db (Session): The database session.
```

**Review by zxenonx — CHANGES_REQUESTED** [review]
Good job.
I left a few comments.
Thanks

File: app/routes/video_routes.py (hunk 1 above)
Comment (DestinedCodes) [reply app/routes/video_routes.py]: I actually used to have conflict with that naming convention and the response object from flask. So it's become a bad habit. I'll stick to good naming convention next time.

**Comment by DestinedCodes** [comment]
Requested changes have been made.

**Review by zxenonx — APPROVED** [review]

**Pull request merged**